docs/known-issues/index.md

```diff
@@ -6,6 +6,40 @@ active investigation by HPE Cray and the wider service.
 
 ## Open Issues
 
+### OOM due to memory leak in libfabric (Added: 2022-02-23)
+
+There is an underlying memory leak in the version of libfabric on ArCHER2 (that comes as part of the underlying
```

Comment:
Capitalisation of ARCHER2 is not correct.

Reply:
Fixed